.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 orbs:
   aws-cli: circleci/aws-cli@2.0.3
-  ruby: circleci/ruby@1.1.2
+  ruby: circleci/ruby@1.2.0
```

Comment:
We had some deprecation warnings in the bundling process that would be fixed by using a newer version of the script that lives in this orb